### ISSUE TITLE
missing readmes

### ISSUE THREAD

**Issue by fedhere** [zem232/PUI2018_zem232#2]
Your homework folders are missing the README! each one should have a readme describing the scope of the homework and who did what in your group (even if you work alone you need the readme)

**Comment by zem232** [comment]
Yes I realize that now! For the first couple homeworks I didn't realize I needed a README file, but I am adding them now (also sorry I didn't see this until now -- I never got a notification)
